### DISCUSSION TITLE
Connection between validation and ignoring of a form element

### DISCUSSION THREAD

**Discussion by fsoffel** [formkit/formkit#710] [Ideas]
I have an app where I have a field whose content I don't want to have in my forms model. But I want field validation for it and in case of invalidity I want to disable my submit button.
I do not see a simple solution for this using FormKit. It seems like the `ignore` property is the way to go. I see the validation error message then but the submit button does not recognize that the validation state it ([https://formkit.com/inputs/form](https://formkit.com/inputs/form) Section "Validity State"). I tried it with dynamically setting `ignore` dynamically if and only if the validation requirements are not met. But the dynamic change to the value of `ignore` does not seem to have an affect on the validity state of the form.

Either `ignore` should not work lazy, or there should be away to only ignore the form field in the model and not disable validation.

**Comment by andrew-boyd** [comment]
Sounds like this would also solve https://github.com/orgs/formkit/discussions/707. We'll look into it.
